CHANGELOG.md

```diff
@@ -64,7 +65,7 @@ The traffic can be influenced even when the user device moves to a different geo
 
 ## Release Notes
 
-This pre-release contains the definition and documentation of:
+This release contains the definition and documentation of:
```

Comment:
r1.2 was actually a pre-release. But also fine with release here, if you prefer that.